mistral/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:trusty
```

Comment:
The recommended way to use `FROM debian:wheezy`, as we do for all `st2` components:  https://github.com/StackStorm/st2-dockerfiles/blob/master/stackstorm/Dockerfile#L1

It's very small by size and it's better to reuse image layers for one OS, instead of adding many.

See:
* https://gist.github.com/armab/fc3f0323c65b709ae52e#st2-dockerfiles-repo
* https://labs.ctl.io/optimizing-docker-images/